ignite/contrib/engines/common.py

```diff
@@ -316,14 +332,20 @@ def setup_tb_logging(output_path, trainer, optimizers=None, evaluators=None, log
         **kwargs: optional keyword args to be passed to construct the logger.
 
     Returns:
-        :class:`~ignite.contrib.handlers.tensorboard_logger.TensorboardLogger`
```

Comment:
Let's keep all those links as links, otherwise url is missing and the text can not be clicked on